test/e2e/specs/importer/importer__import-hosted-site.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * @group calypso-pr
+ */
+import { DataHelper, StartImportFlow, TestAccount } from '@automattic/calypso-e2e';
+import { Browser, Page } from 'playwright';
+import type { TestAccountName } from '@automattic/calypso-e2e';
+
+declare const browser: Browser;
+
+describe( DataHelper.createSuiteTitle( 'Stepper: setup/import-hosted-site' ), () => {
+	let page: Page;
+	let startImportFlow: StartImportFlow;
+
+	beforeAll( async () => {
+		page = await browser.newPage();
+		startImportFlow = new StartImportFlow( page );
+	} );
+
+	const loginAsUser = ( user: TestAccountName ) => {
+		it( `Login as ${ user }`, async () => {
+			const testAccount = new TestAccount( user );
+			await testAccount.authenticate( page );
+		} );
+	};
+
+	const navigateToImportHostedSiteFlow = ( step: string, siteSlug?: string, from?: string ) => {
+		it( `Navigate to page setup/import-focused?from=${ from }`, async () => {
```

Comment:
should be import-hosted-site here?

Reply:
Funny, I've fixed this but forgot to push the change. :)